content/en/docs/concepts/configuration/liveness-readiness-startup-probes.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Liveness, Readiness, and Startup Probes
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+Pending
+
+<!-- body -->
+
+## Startup Probe
+
+A startup probe verifies whether the application within a container is started. 
+
+Liveness and readiness probes do not start until it succeeds. If a container fails its startup probe, the container is killed and follows the pod’s restartPolicy.
+
+This type of probe is only executed at startup, unlike readiness probes, which are run periodically.
+
+## Readiness Probe
+
+Readiness probes determine whether a container is ready to serve traffic. This is useful when waiting for an application to perform time-consuming initial tasks, such as establishing network connections, loading files, and warming caches. 
+
+If the readiness probe returns a failed state, Kubernetes will remove the pod from all matching service endpoints.
+
+Readiness probes runs on the container during its whole lifecycle.
+
+## Liveness Probe
+
+Liveness probes determine whether an application running in a container is in a healthy state. For example, liveness probes could catch a deadlock, where an application is running, but unable to make progress.
+
+If the liveness probe detects an unhealthy state, then Kubernetes kills the container and tries to redeploy it.
+
+Liveness probes do not wait for readiness probes to succeed. If you want to wait before executing a liveness probe you should use initialDelaySeconds or a startupProbe.
+
+You can configure liveness probe with the following methods:
+
+* Command
+* HTTP Request
+* TCP Socket
+* gRPC Protocol
+
+Read about [Configure Liveness, Readiness and Startup Probes](/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes).
```

Comment:
Some feedbacks:

- You may need a English native speaker to help review the language;
- You may actually migrate some texts about basic concepts from the task page;
- You may want to emphasis on the use cases, the problems that are targeted by
  each type of probes, the design considerations behind the implementation that
  users may need to be aware of.

Reply:
Also, please follow the structure for a concept page.

Reply:
Sure.
I will make another version.

Reply:
FYI. I have just proposed a fix to the existing probe task (https://github.com/kubernetes/website/pull/42619).